### PR TITLE
netcore2.0 migration

### PR DESCRIPTION
**Pull request by Dongata**
## Background 
In order to use some tools, we first need to update our version of the framework for compatibility reasons

## Changes done
- updates cs projects
- updates nuget references
- changes deprecated code

### CODE REVIEW

File: WebApiCoreSeed.WebApi/WebApiCoreSeed.WebApi.csproj [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="1.1.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="1.1.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.2.1" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="2.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.0.3" />
```
Comment (matiasbeckerle) [inline WebApiCoreSeed.WebApi/WebApiCoreSeed.WebApi.csproj]: Hmm this is a startup project. We should not prepick a db provider other than in-memory or sqlite. This is not related with your changes anyway. You can ignore this comment.

Comment (Dongata) [reply WebApiCoreSeed.WebApi/WebApiCoreSeed.WebApi.csproj]: you're right. I see you created an issue regarding that, so I will attack it on other pr
Ty c:
